src/resizing.jl

```diff
@@ -262,7 +263,7 @@ julia> imresize(img, (128, 128)) # 128*128
 julia> imresize(img, (1:128, 1:128)) # 128*128
 julia> imresize(img, (1:128, )) # 128*256
 julia> imresize(img, 128) # 128*256
-julia> imresize(img, ratio = 0.5) # 
+julia> imresize(img, ratio = 0.5) #
```

Comment:
This seems to be a mistake in previous PRs. Could you help add the size information here?

Reply:
Sure. Added in the second commit. Just 128*128.